core/src/test/java/com/novoda/merlin/service/HostPingerTest.java

```diff
@@ -10,7 +10,7 @@
 import static org.mockito.MockitoAnnotations.initMocks;
 
 @RunWith(JUnit4.class)
```

Comment:
As a boyscouting `@RunWith(JUnit4.class)` annotation can be removed, right? Can be done later.